src/mapeo-project.js

```diff
@@ -840,6 +849,36 @@ async function deleteAll(dataType) {
   return Promise.all(deletions)
 }
 
+/**
+ * @param {Logger} logger
+ */
+function deleteTranslations(logger) {
+  /**
+   * @param {Object} dataTypes
+   * @param {MapeoProject['translation']} dataTypes.translation
+   * @param {MapeoProject['preset']} dataTypes.presets
+   * @param {MapeoProject['field']} dataTypes.fields
+   */
+  return async function (dataTypes) {
```

Comment:
This "function that returns a function" feels unnecessary. Could you just pass the `Logger` as an argument? Or is there a reason for this closure?

Reply:
yeah, reverted on last commit